x-pack/plugins/security_solution/public/common/components/threat_match/entry_item.tsx

```diff
@@ -55,11 +60,27 @@ export const EntryItem: React.FC<EntryItemProps> = ({
     [onChange, entry]
   );
 
+  const dataViewInstance = useMemo(() => {
+    if (indexPattern != null) {
+      const dv = new DataView({ spec: indexPattern, fieldFormats });
+      return { ...dv, fields: Object.values(indexPattern.fields ?? {}) } as DataView;
+    }
+    return [];
+  }, [indexPattern, fieldFormats]);
+
+  const threatDataViewInstance = useMemo(() => {
+    if (threatIndexPatterns != null) {
+      const dv = new DataView({ spec: threatIndexPatterns, fieldFormats });
+      return { ...dv, fields: Object.values(threatIndexPatterns.fields ?? {}) } as DataView;
+    }
+    return [];
```

Comment:
nit: separate transform function can be added to DataView instance from index and fields formats. dataViewInstance and threatDataViewInstance look indentical